### ISSUE TITLE
`RuntimeError` in multichannel IDTCWT

### ISSUE THREAD

**Issue by lostanlen** [kymatio/murenn#20]
I think that our implementation of `INV_J1.forward` has a bug when passing a multichannel input.

Here's how i found the problem: (note the `ch=3` instead of more customary `ch=1`)

```python
import murenn
import torch

N = 2**10
J = 8
ch = 3

x = torch.zeros(1, ch, N)
dtcwt = murenn.DTCWT(J=J)
idtcwt = murenn.DTCWTInverse(J=J)
x_phi, x_psis = dtcwt(x)
y = idtcwt((x_phi, x_psis))
```

Error message:

```python
File ~/MuReNN/murenn/murenn/murenn/dtcwt/transform_funcs.py:178, in INV_J1.forward(ctx, lo, hi_r, hi_i, g0, g1, padding_mode)
    175 ctx.mode = mode_to_int(padding_mode)
    177 # Apply dual low-pass filtering
--> 178 x0 = torch.nn.functional.conv1d(pad_(lo, g0, padding_mode), g0_rep)
    180 # Apply dual high-pass filtering
    181 hi = torch.stack((hi_r, hi_i), dim=-1).view(b, ch, T)
```

```
Given groups=1, weight of size [7, 1, 7], expected input[1, 7, 8198] to have 1 channels, but got 7 channels instead
```

**Comment by xir4n** [comment]
Thanks for the remark! It seems I forgot to specify “groups=ch” when calling the conv1d function...

**Comment by lostanlen** [comment]
closed by #22
